CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 
+### Breaking Changes
+- Bump sensu-plugin version to 4.0 for Sensu 1.x to Sensu Go event conversion
```

Comment:
To better help people evaluate upgrades where we break something its best to link to relevant changelog entries, for example: https://github.com/sensu-plugins/sensu-plugins-windows/pull/76/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR15 not all will have this for example dropping ruby support.